articles/healthcare-apis/iot/iot-connector-overview.md

```diff
@@ -17,9 +17,12 @@ ms.author: jasteppe
 
 ## Overview
 
-IoT connector is an optional service of the Azure Healthcare APIs that provides the capability to ingest message data from Internet of Medical Things (IoMT) devices and wearables. 
+IoT connector is an optional service of the Azure Healthcare APIs designed to ingest health data from multiple and disparate Internet of Medical Things (IoMT) devices and persisting the health data in a FHIR server.
 
-IoT connector transforms device data into Fast Healthcare Interoperability Resources (FHIR&#174;)-based [Observation](https://www.hl7.org/fhir/observation.html) resources and then persists the transformed messages into the Azure Healthcare APIs [FHIR service](../fhir/overview.md).
+The connector is important since health data collected from patients and health care consumers can be fragmented from access across multiple systems, device types, and formats. Managing healthcare data can be difficult, however, trying to gain insight from the data can be one of the biggest barriers to population and personal wellness understanding as well as sustaining health.  
```

Comment:
Maybe change from "The connector" to "IoT connector"? The way I am reading this, seems a bit confusing "The connector is important since health data collected from patients and health care consumers can be fragmented from access across multiple systems, device types, and format". Is access to the data fragmented? I am just throwing this out there as an example "IoT connector is important since health data collected from patients and health care consumers can be fragmented due to inputs from across multiple systems, device types, and formats"

Reply:
Will add IoT connector in. 
What is an input specifically? 
The main issue is Bob's data is all messed up from his  patient generated health data PGHD from his watch, outpatient and pcp office and last admission; ie med list, follow up, dc instructions.